src/components/carbon-provider/carbon-provider.mdx

```diff
@@ -27,7 +24,7 @@ You can import the provider and any intended themes like so:
 
 ```ts
 import CarbonProvider from "carbon-react/lib/components/carbon-provider";
-import { sageTheme, mintTheme } from "carbon-react/lib/style/themes";
+import { sageTheme, mintTheme } from "carbon-react/lib/styles/themes";
```

Comment:
```suggestion
import { sageTheme, mintTheme } from "carbon-react/lib/style/themes";
```

**comment** - Maybe I'm missing something but I'm not sure why you've changed this - I was going to praise you for spotting a tiny correction in the midst of this huge PR, but on checking the repo the folder is definitely called `style` rather than `styles`.